2-how-node-works/starter/events.js

```diff
@@ -0,0 +1,42 @@
+const EventEmitter = require(`events`);
+const http = require(`http`);
+
+class Sales extends EventEmitter {
+  constructor() {
+    super();
```

Comment:
Can you consider removing this explicit constructor? To my knowledge, a constructor will always call the super constructor by default. If this also happens with Node, there is no need to override the constructor